website/versioned_docs/version-24.6/JestObjectAPI.md

```diff
@@ -0,0 +1,626 @@
+---
+id: version-24.6-jest-object
```

Comment:
I don't think you have to add a new one - not sure though!

Reply:
I wasn't sure either, but the other files in `versioned_docs` do so I added it just in case :). I'm happy to change it if it's not ideal.

Reply:
Every PR is deployed, so you can remove this and check the deployed site on netlify


https://deploy-preview-8550--jest-preview.netlify.com/

If the versioned doc is incorrect, you can re-add this

Reply:
Looks like what you need to do is edit 22.x, 23.x and 24.0 versions of JestObjectAPI, separate 24.6 is not necessary

Reply:
Ah that was a great idea. Editing the previous versions added the example there too which is still relevant for those versions. 👍 

I also removed the file `JestObjectAPI` from the versioned_docs for 24.6, as it's not needed anymore like you mentioned.

I tested locally (with `yarn start`) and the changes are appearing for all versions.